src/ResourceManager/Network/Commands.Network/Commands.Network.csproj

```diff
@@ -61,6 +61,7 @@
     <Reference Include="Microsoft.Azure.Management.Network, Version=19.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.5.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL" />
```

Comment:
I don't think that we should need to change Commands.Network or Commands.Profile . Were these changes intentional?

Reply:
it was auto-generated due to the new package version. reverting now.